plaso/filters/parser_filter.py

```diff
@@ -181,33 +188,41 @@ def ExpandPresets(self, presets_manager, expression):
     return self._JoinExpression(excludes, includes)
 
   def SplitExpression(self, expression):
-    """Determines the excluded and included parsers from an expression string.
+    """Determines the excluded and included values from an expression string.
+
+    Note that presets will not be expanded.
```

Comment:
> Note that presets will not be expanded.

Note that function will not expand presets and preset names are treated like parser names.